mlir/test/Dialect/LLVM/lower-to-llvm-e2e-with-top-level-named-sequence.mlir

```diff
@@ -1,12 +1,12 @@
 // Note: We run CSE here to make the pattern matching more direct.
 
-// RUN: mlir-opt %s -test-lower-to-llvm -cse | FileCheck %s
+// RUN: mlir-opt %s -test-lower-to-llvm -cse -canonicalize | FileCheck %s
```

Comment:
Is this for convenience/not needing to update test or another reason?

Reply:
Yes, this was just the easiest way to fix the test case. `-reconcile-unrealized-casts` used to fold some ops, and the new implementation no longer does not.